pilot/pkg/model/push_context.go

```diff
@@ -135,11 +135,13 @@ type sidecarIndex struct {
 	// computedSidecarsByNamespace contains the default sidecar for namespaces that do not have a sidecar.
 	// These may be DefaultSidecarScopeForNamespace if rootConfig is empty or ConvertToSidecarScope if not.
 	// These are lazy-loaded. Access protected by defaultSidecarMu
+	// +checklocks:defaultSidecarMu
```

Comment:
what;s this for

Reply:
didn't mean to merge this.. was experimenting with something else